GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -75,7 +75,9 @@ GDTCORNetworkMobileSubtype GDTCORNetworkMobileSubTypeMessage() {
   if (@available(iOS 12, *)) {
     NSDictionary<NSString *, NSString *> *networkCurrentRadioAccessTechnologyDict =
         networkInfo.serviceCurrentRadioAccessTechnology;
-    if (networkCurrentRadioAccessTechnologyDict) {
+    if (networkCurrentRadioAccessTechnologyDict.count) {
+      // In iOS 12, multiple radio technologies can be captured. We prefer not particular radio
+      // tech to another, so we'll just return the first value in the dictionary.
       networkCurrentRadioAccessTechnology = networkCurrentRadioAccessTechnologyDict.allValues[0];
```

Comment:
This needs to be allKeys, not allValues

Reply:
The keys can be associated with other apis, like serviceSubscriberCellularProviders dictionary, but for the networkCurrentRadioAccessTechnology string we want is stored in the values.